pycm/pycm_output.py

```diff
@@ -276,11 +269,7 @@ def html_end(version):
     :type version:str
     :return: html_end as str
     """
-    result = "</body>\n"
-    result += '<p style="text-align:center;border-top:1px solid black;">Generated By ' \
-        '<a href="http://www.pycm.ir" ' \
-              'style="text-decoration:none;color:red;">PyCM</a> Version ' + version + '</p>\n'
-    result += "</html>"
+    result = HTML_END_TEMPLATE.format(version)
     return result
```

Comment:
Here too.

Reply:
Fixed in https://github.com/sepandhaghighi/pycm/pull/368/commits/fb72bbf1568c129a5e48284386c2aa66bc48ac1f